types/api.d.ts

```diff
@@ -13,11 +13,12 @@ export type TokenDictionary = {
 export interface EvidenceRequestResponse
   extends Omit<
     IEvidenceRequest,
-    'createdAt' | 'deliveryMethods' | 'documentTypes'
+    'createdAt' | 'deliveryMethods' | 'documentTypes' | 'userRequestedBy'
```

Comment:
I don't think we need to include this in the `Omit` list. I think we only omit fields which are optional, or additional parsing needs to be done - for example `createdAt` we convert to `DateTime` in `response-mapper.ts`

Reply:
👍